include/soci/ref-counted-statement.h

```diff
@@ -56,6 +57,7 @@ class SOCI_DECL ref_counted_statement_base
 
     template <typename T>
     void accumulate(T const & t) { get_query_stream() << t; }
+    inline void accumulate(std::wstring const & t) { get_query_stream() << wide_to_utf8(t); }
```

Comment:
Very minor, but `inline` here is redundant and inconsistent, so
```suggestion
    void accumulate(std::wstring const & t) { get_query_stream() << wide_to_utf8(t); }
```

Reply:
True